pkgs/development/python-modules/thorlabspm100/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, pytest-runner
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  version = "1.2.2";
+  pname = "ThorlabsPM100";
```

```suggestion
  pname = "thorlabspm100";
```
`pname` should be normalized according to [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names).
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-contributing